tests/ops/test_qubit_ops.py

```diff
@@ -509,7 +508,7 @@ def test_adjoint_with_decomposition(self, op_builder):
         decomposed_ops = op.decomposition(wires=op.wires)
         with qml.tape.QuantumTape() as adjoint_tape:
             qml.adjoint(op_builder)()
-        for a, b in zip(decomposed_ops, reversed(adjoint_tape.operations)):
+        for a, b in zip(decomposed_ops, reversed(adjoint_tape.expand().operations)):
```

Comment:
How come this was required?

Reply:
It is related to Josh's comment below, before the direct implementation of adjoint for QFT, the operation was decomposed and adjoint was taken for each one. But now we need to expand it before taking `.operations`.